sys/net/gnrc/network_layer/sixlowpan/frag/rbuf.c

```diff
@@ -48,6 +48,9 @@ static rbuf_t rbuf[RBUF_SIZE];
 
 static char l2addr_str[3 * IEEE802154_LONG_ADDRESS_LEN];
 
+static xtimer_t _gc_timer;
+static msg_t _gc_timer_msg;
```

Comment:
please set the msg `type` here directly with a static initialization 
```c
static msg_t _gc_timer_msg = { .type = GNRC_SIXLOWPAN_MSG_FRAG_GC_RBUF };
```

instead of setting it everytime in
```c
static inline void _set_rbuf_timeout(void) { ... }
```
below.

Reply:
True. Thought I could save some memory with that, but I was wrong and am wasting memory and time with it.

Reply:
Done